bananas_api/new_upload/classifiers/heightmap.py

```diff
@@ -0,0 +1,76 @@
+from ...helpers.api_schema import Classification
+from ...helpers.enums import (
+    TerrainType,
+    Resolution,
+    Shape,
+)
+from ..readers.heightmap import Heightmap
+
+
+def classify_heightmap(heightmap: Heightmap) -> Classification:
+    classification = {}
+
+    surface = heightmap.size[0] * heightmap.size[1]
+    if surface < 512000:
+        classification["resolution"] = Resolution.STANDARD
+    else:
+        classification["resolution"] = Resolution.HIGH
+
+    aspect_ratio = heightmap.size[0] / heightmap.size[1]
+    if aspect_ratio < 1:
+        aspect_ratio = 1 / aspect_ratio
+
+    if aspect_ratio == 1:
```

Comment:
`aspect_ratio` is a float, so instead of "== 1" and "== 2" I would expect "< 1.2" and "< 2.5".

Reply:
Yeah, most maps aren't like this, but it is a good point: some have weird dimensions. Tnx!